src/artofillusion/polymesh/Command.java

```diff
@@ -22,7 +23,9 @@
      * execute the command
      *
      */
-    public void execute();
+    public default void execute() {
+      redo();
```

Comment:
I don't think this makes sense. I'd have to go through a lot of code to see why so many of the implementations of `execute()` just call `redo()`, but this is not an appropriate default.

Reply:
I think all this PR has no sense without this change

Reply:
That may be. I'm trying to understand why the PME is using it's own undo/redo system here.